sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -86,6 +87,20 @@ class EventHubConsumerClient(ClientBase):
     :paramtype checkpoint_store: ~azure.eventhub.CheckpointStore
     :keyword float load_balancing_interval: When load-balancing kicks in. This is the interval, in seconds,
      between two load-balancing evaluations. Default is 10 seconds.
+    :keyword float partition_ownership_expiration_interval: A partition ownership will expire after this number
+     of seconds. Every load-balancing evaluation will automatically extend the ownership expiration time.
+     Default is 6 * load_balancing_interval.
```

Comment:
Might be good to be super explicit and finish this sentence with `"(i.e. 60 seconds when using the default load_balancing_interval of 10 seconds)"`

Reply:
Also - did we confirm the default behaviour as being a multiple?

Reply:
Updated the docstring as suggested.
Kyle said there were no video available yet. I see js is 60, java is *6, .net is 30. I'll confirm with them again.

Reply:
Discussed across languages. I don't think we can have a consensus quickly.
Let's keep this logic and continue the discussion in next release.